source/install/desktop.rst

```diff
@@ -35,6 +35,20 @@ Windows 10+, Windows 8.1+, Windows 7+
 
 This will start an installer for the app. Once finished, the Mattermost desktop app will open automatically.
 
+Group Policies
+~~~~~~~~~~~~~~~~~~~~~
+Copy the ``mattermost.admx`` from ``%INSTALLDIR\gpo_template`` to ``C:\Windows\PolicyDefinitions`` or if available ``\\FQDNDomain\sysvol\FQDNDomain\Policies\PolicyDefinitions``
```

Comment:
Is this normal the install directory variable is specified as `%INSTALLDIR` without a trailing `%`?

Reply:
Fixed :/